lmeval.py

```diff
@@ -51,7 +51,7 @@ def parse_args():
     parser.add_argument("--device", type=str, default="cuda:0")
     parser.add_argument("--output_path", default=None)
     parser.add_argument("--limit", type=int, default=None)
-    parser.add_argument("--no_cache", action="store_true")
+    parser.add_argument("--no_cache", action="store_false")
```

Comment:
no_cache is recommended to be set true in the tests. Also caching is not compatible with evaluating pre-loaded models. Now it is set to True by default for convenience.